oarepo_oaipmh_harvester/resources/records/resource.py

```diff
@@ -22,6 +25,14 @@ def create_url_rules(self):
     @request_view_args
     @response_handler(many=True)
     def execute(self, *args, **kwargs):
```

Comment:
missing any kind of security here (anyone can call this as many times as they want, flooding celery with harvest tasks). There must be an Admin action permission guard to call this endpoiint: https://inveniordm.docs.cern.ch/develop/topics/administration_panel/#admin-actions-permissions